File: python_vs_golang/plusminus.go
package main

import "fmt"

func plusMinus(arr []int32) {
    // Write your code here
    pos := float64(0)
    neg := float64(0)
    zero := float64(0)
    for _, num := range arr {
        if num > 0 {
            pos++
        } else if num < 0 {
            neg++
        } else if num == 0 {
            zero++
        }
    }  
    div := float64(len(arr))
    fmt.Println(pos/div)
    fmt.Println(neg/div)
    fmt.Println(zero/div)
}

File: plusminus.py
def plusMinus(arr):
    # Write your code here
    pos = 0
    neg = 0
    zero = 0
    for num in arr:
        if num < 0:
            neg += 1
        elif num > 0:
            pos += 1
        elif num == 0:
            zero += 1
    print(pos/len(arr))
    print(neg/len(arr))
    print(zero/len(arr))
